Dockerfile

```diff
@@ -1,6 +1,13 @@
-FROM gcr.io/distroless/static
+FROM golang:1.14 as builder
+ARG GOARCH=amd64
+ENV GOARCH=${GOARCH}
+WORKDIR /go/src/k8s.io/kube-state-metrics/
+COPY . /go/src/k8s.io/kube-state-metrics/
```

Comment:
Since you've already set the workdir, shouldn't this do?
```suggestion
COPY . .
```

Reply:
Thanks for the feedback, added it!